### DISCUSSION TITLE
click button in header section of list report page

### DISCUSSION THREAD

**Discussion by diya-dhan** [ui5-community/wdi5#461] [Q&A]
Hi,

I am trying to execute the 'Go' button in the header part of my list report page of my fiori app. According to the test library documentation, I need to execute the iExecuteAction() of the onHeader class. But this method requires an ActionIdentifier parameter. 

How do I provide the identifier to the function?

Thanks in advance.

**Comment by vobu** [comment]
Hi Diya,
this is probably less of a discussion than a support request.
Can you please put it in the appropriate category?
And at the same time, please provide as much context as possible, by using the Issue template we provide. Only then can we not guesstimate 🔮, but actually help.
Thanks!

**Comment by diya-dhan** [comment]
Hey Volker,

Yes I'll put it in relevant category. Just that I can't find something like support request under discussions? Is it under another section or else would it more apt raise an issue?

Thanks a lot
